src/main/java/org/folio/processor/translations/TranslationsFunctionHolder.java

```diff
@@ -84,6 +85,30 @@ public String apply(String identifierValue, int currentIndex, Translation transl
       return StringUtils.EMPTY;
     }
   },
+  SET_LOCATION() {
```

Comment:
TranslationsFunctionHolder already contains "SET_LOCATION"

Reply:
Yes, please remove it, I created new location function to get all related reference data, I'm also going to create quick fix for mod-data-export to make build green

Reply:
It wasn't added new in this PR, it was already existing, I think it didn't show it as merge conflict from master, fixed that now